docs/architecture/adr-071-proposer-based-timestamps.md

```diff
@@ -281,15 +254,15 @@ Validators already check that +2/3 prevotes were seen in `POLRound`, so this doe
 A validator will also check that the proposed timestamp is greater than the timestamp of the block for the previous height.
```

Comment:
A validator prevotes `nil` for a block when it does not receive the proof-of-lock prevotes before the propose timeout.